pkg/question/question.go

```diff
@@ -382,39 +394,68 @@ func AskInstanceType(h *ec2helper.EC2Helper, defaultInstanceType string) (*strin
 		}
 	}
 
-	question := "Instance Type (eg. m5.xlarge, c5.xlarge)"
+	question := "Which instance type should by used? (eg. m5.xlarge, c5.xlarge)"
```

Comment:
Typo: `by` -> `be`. But I'd prefer to rewrite with active voice. Some ideas:

* Enter an instance type
* Which instance type should we use?
* Which instance type do you want?

Reply:
I'll address this in the next PR when I reformat the questions